gatsby/src/components/BackToTop/BackToTop.tsx

```diff
@@ -0,0 +1,42 @@
+import React, { useEffect, useState } from "react";
+
+import styles from "./BackToTop.module.scss";
+
+interface BackToTopProps {
+	scrollTo?: string;
+}
+
+export const BackToTop: React.FC<BackToTopProps> = ({
+	scrollTo = "content-start",
+}: BackToTopProps) => {
+	const [isFixed, setIsFixed] = useState(false);
+
+	function handleScroll() {
```

Comment:
Should we useCallback this? Because otherwise the function won't be the same reference when we removeEventListener? Or maybe it will but the function will get recreated each render.